src/scss/_vars.scss

```diff
@@ -104,7 +104,7 @@ $gutter-size: 20px;
 // Responsive Breakpoints
 // key order matters (must be from smallest to largest breakpoint)
 $screen: (
-  xs: 0em,
+  xs: 0,
```

Comment:
removed unnecessary `em` unit (`0em` is the same as `0px` which is the same as `0`)

per [`length-zero-no-unit`](https://stylelint.io/user-guide/rules/length-zero-no-unit) configuration

Reply:
I'm going to update this file to make an exception for this SCSS map value.